core/src/mindustry/world/blocks/production/Drill.java

```diff
@@ -287,6 +294,15 @@ public void updateTile(){
 
                 drillEffect.at(getX() + Mathf.range(size), getY() + Mathf.range(size), dominantItem.color);
             }
+
+            
```

Comment:
unnecessary empty line

Reply:
All right, I made a commit to remove extra whitespace